### PR TITLE
added skill format details

### PR DESCRIPTION
**Pull request by saurabhjn76**
Fixes issue #424 

Changes: added format for the skill as discussed in [https://github.com/fossasia/susi_skill_data/issues/254](https://github.com/fossasia/susi_skill_data/issues/254).
```
::name <Skill_name>
::author <author_name>
::author_url <author_url>
::description <description> 
::dynamic_content <Yes/No>
::developer_privacy_policy <link>
::image <image_url>
::term_of_use <link>
#Intent
User query1|query2|quer3....
!example:<The question that should be shown in public skill displays>
!expect:<The answer expected for the above example>
Answer for the user query
```

Screenshots for the change: N/A

### CODE REVIEW

File: src/ai/susi/mind/SusiMind.java [hunk 1]
```diff
@@ -214,6 +251,19 @@ public SusiMind learn(JSONObject json, File origin) {
...
+                if(json.has("skill_name"))
+                    this.skillNames.put(intent.getSkill(), json.getString("skill_name"));
+                if(json.has("author"))
+                    this.authors.put(intent.getSkill(), json.getString("author"));
+                if(json.has("author_url"))
+                    this.authorsUrl.put(intent.getSkill(), json.getString("author_ur;"));
+                if(json.has("developer_privacy_policy"))
+                    this.devloperPrivacyPolicies.put(intent.getSkill(), json.getString("developer_privacy_policy"));
+                if(json.has("term_of_use"))
+                    this.termsOfUse.put(intent.getSkill(), json.getString("term_of_use"));
```
Comment (mariobehling) [inline src/ai/susi/mind/SusiMind.java]: it is "terms of use". Change to plurals everywhere!

Comment (saurabhjn76) [inline src/ai/susi/mind/SusiMind.java]: ok :+1:

**Comment by mariobehling** [comment]
Please use the same standard everywhere:
* skill_name instead of Skill_name
* terms_of_use instead of term_of_use

**Comment by mariobehling** [comment]
Thank you!

**Pull request merged**